<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1104</width>
    <height>786</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>951</width>
     <height>631</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>produits physiques</string>
    </attribute>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1101</width>
       <height>731</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#frame{
background-color: #333;
border-radius:15px;

}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QTableView" name="tabF">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>120</y>
        <width>661</width>
        <height>391</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>20</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Cinn">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>119</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-width: 1px; border-radius: 4px;
font-size: 18px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;
	color: #a9b7c6;
	background:gray;
	selection-background-color:#007b50;
	selection-color: #FFFFFF;</string>
      </property>
      <property name="placeholderText">
       <string>id</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Nom">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>169</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-width: 1px; border-radius: 4px;
font-size: 18px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;
	color: #a9b7c6;
	background:gray;
	selection-background-color:#007b50;
	selection-color: #FFFFFF;</string>
      </property>
      <property name="placeholderText">
       <string>nom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Prenom">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>219</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-width: 1px; border-radius: 4px;
font-size: 18px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;
	color: #a9b7c6;
	background:gray;
	selection-background-color:#007b50;
	selection-color: #FFFFFF;</string>
      </property>
      <property name="placeholderText">
       <string>marque</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="Mail">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>269</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-width: 1px; border-radius: 4px;
font-size: 18px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;
	color: #a9b7c6;
	background:gray;
	selection-background-color:#007b50;
	selection-color: #FFFFFF;</string>
      </property>
      <property name="placeholderText">
       <string>prix</string>
      </property>
     </widget>
     <widget class="QPushButton" name="valider">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>450</y>
        <width>201</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#valider{

background-color: red;
border-radius: 4px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
font-family: century gothic;
font-size: 18px;
color: #a9b7c6;
}
#valider:hover{
border-style: solid;
	border-top-color: transparent;
	border-right-color: transparent;
	border-left-color: transparent;
	border-bottom-color: #e67e22;
	border-bottom-width: 1px;
	border-bottom-radius: 6px;
	border-style: solid;
	color: #FFFFFF;
	padding-bottom: 2px;
	background-color: green;
}</string>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="valider_2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>530</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#valider_2{

background-color: red;
border-radius: 4px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
font-family: century gothic;
font-size: 18px;
color: #a9b7c6;
}
#valider_2:hover{
border-style: solid;
	border-top-color: transparent;
	border-right-color: transparent;
	border-left-color: transparent;
	border-bottom-color: #e67e22;
	border-bottom-width: 1px;
	border-bottom-radius: 6px;
	border-style: solid;
	color: #FFFFFF;
	padding-bottom: 2px;
	background-color: green;
}</string>
      </property>
      <property name="text">
       <string>imprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ModifierF">
      <property name="geometry">
       <rect>
        <x>444</x>
        <y>530</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#ModifierF{

background-color: red;
border-radius: 4px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
font-family: century gothic;
font-size: 18px;
color: #a9b7c6;
}
#ModifierF:hover{
border-style: solid;
	border-top-color: transparent;
	border-right-color: transparent;
	border-left-color: transparent;
	border-bottom-color: #e67e22;
	border-bottom-width: 1px;
	border-bottom-radius: 6px;
	border-style: solid;
	color: #FFFFFF;
	padding-bottom: 2px;
	background-color: green;
}</string>
      </property>
      <property name="text">
       <string>Modifer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SupprimerF">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>530</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#SupprimerF{

background-color: red;
border-radius: 4px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
font-family: century gothic;
font-size: 18px;
color: #a9b7c6;
}
#SupprimerF:hover{
border-style: solid;
	border-top-color: transparent;
	border-right-color: transparent;
	border-left-color: transparent;
	border-bottom-color: #e67e22;
	border-bottom-width: 1px;
	border-bottom-radius: 6px;
	border-style: solid;
	color: #FFFFFF;
	padding-bottom: 2px;
	background-color: green;
}</string>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="valider_3">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>70</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#valider_3{

background-color: red;
border-radius: 4px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
font-family: century gothic;
font-size: 18px;
color: #a9b7c6;
}
#valider_3:hover{
border-style: solid;
	border-top-color: transparent;
	border-right-color: transparent;
	border-left-color: transparent;
	border-bottom-color: #e67e22;
	border-bottom-width: 1px;
	border-bottom-radius: 6px;
	border-style: solid;
	color: #FFFFFF;
	padding-bottom: 2px;
	background-color: green;
}</string>
      </property>
      <property name="text">
       <string>trier par prix</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="cinS">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>529</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-width: 1px; border-radius: 4px;
font-size: 18px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;
	color: #a9b7c6;
	background:gray;
	selection-background-color:#007b50;
	selection-color: #FFFFFF;</string>
      </property>
      <property name="placeholderText">
       <string>id</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>330</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout"/>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>471</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-family: century gothic;
font-size: 30px;
color: #a9b7c6;</string>
      </property>
      <property name="text">
       <string>Gestion des produits physiques</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>330</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-family: century gothic;
font-size: 18px;
color: #a9b7c6;</string>
      </property>
      <property name="text">
       <string>disponibilité</string>
      </property>
     </widget>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>-90</y>
        <width>661</width>
        <height>80</height>
       </rect>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>70</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-width: 1px; border-radius: 4px;
font-size: 18px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;
	color: #a9b7c6;
	background:gray;
	selection-background-color:#007b50;
	selection-color: #FFFFFF;</string>
      </property>
      <property name="placeholderText">
       <string>Donner l'id</string>
      </property>
     </widget>
     <widget class="QPushButton" name="valider_5">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>70</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#valider_5{

background-color: red;
border-radius: 4px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
font-family: century gothic;
font-size: 18px;
color: #a9b7c6;
}
#valider_5:hover{
border-style: solid;
	border-top-color: transparent;
	border-right-color: transparent;
	border-left-color: transparent;
	border-bottom-color: #e67e22;
	border-bottom-width: 1px;
	border-bottom-radius: 6px;
	border-style: solid;
	color: #FFFFFF;
	padding-bottom: 2px;
	background-color: green;
}</string>
      </property>
      <property name="text">
       <string>chercher</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>330</y>
        <width>48</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>non</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>330</y>
        <width>98</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>oui</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>produits chimiques</string>
    </attribute>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1101</width>
       <height>731</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#frame_2{
background-color: #333;
border-radius:15px;

}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLineEdit" name="CinnC">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>119</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-width: 1px; border-radius: 4px;
font-size: 18px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;
	color: #a9b7c6;
	background:gray;
	selection-background-color:#007b50;
	selection-color: #FFFFFF;</string>
      </property>
      <property name="placeholderText">
       <string>id</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="NombreA">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>320</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-width: 1px; border-radius: 4px;
font-size: 18px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;
	color: #a9b7c6;
	background:gray;
	selection-background-color:#007b50;
	selection-color: #FFFFFF;</string>
      </property>
      <property name="placeholderText">
       <string>quantité</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="NumeroC">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>269</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-width: 1px; border-radius: 4px;
font-size: 18px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;
	color: #a9b7c6;
	background:gray;
	selection-background-color:#007b50;
	selection-color: #FFFFFF;</string>
      </property>
      <property name="placeholderText">
       <string>prix</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="PrenomC">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>219</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-width: 1px; border-radius: 4px;
font-size: 18px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;
	color: #a9b7c6;
	background:gray;
	selection-background-color:#007b50;
	selection-color: #FFFFFF;</string>
      </property>
      <property name="placeholderText">
       <string>marque</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="NomC">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>169</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-width: 1px; border-radius: 4px;
font-size: 18px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;
	color: #a9b7c6;
	background:gray;
	selection-background-color:#007b50;
	selection-color: #FFFFFF;</string>
      </property>
      <property name="placeholderText">
       <string>nom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="cinSC">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>529</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-width: 1px; border-radius: 4px;
font-size: 18px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;
	color: #a9b7c6;
	background:gray;
	selection-background-color:#007b50;
	selection-color: #FFFFFF;</string>
      </property>
      <property name="placeholderText">
       <string>id</string>
      </property>
     </widget>
     <widget class="QTableView" name="tabC">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>120</y>
        <width>661</width>
        <height>391</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="ModifierC">
      <property name="geometry">
       <rect>
        <x>444</x>
        <y>530</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#ModifierC{

background-color: red;
border-radius: 4px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
font-family: century gothic;
font-size: 18px;
color: #a9b7c6;
}
#ModifierC:hover{
border-style: solid;
	border-top-color: transparent;
	border-right-color: transparent;
	border-left-color: transparent;
	border-bottom-color: #e67e22;
	border-bottom-width: 1px;
	border-bottom-radius: 6px;
	border-style: solid;
	color: #FFFFFF;
	padding-bottom: 2px;
	background-color: green;
}</string>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SupprimerC">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>530</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#SupprimerC{

background-color: red;
border-radius: 4px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
font-family: century gothic;
font-size: 18px;
color: #a9b7c6;
}
#SupprimerC:hover{
border-style: solid;
	border-top-color: transparent;
	border-right-color: transparent;
	border-left-color: transparent;
	border-bottom-color: #e67e22;
	border-bottom-width: 1px;
	border-bottom-radius: 6px;
	border-style: solid;
	color: #FFFFFF;
	padding-bottom: 2px;
	background-color: green;
}</string>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SupprimerC_2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>530</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#SupprimerC_2{

background-color: red;
border-radius: 4px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
font-family: century gothic;
font-size: 18px;
color: #a9b7c6;
}
#SupprimerC_2:hover{
border-style: solid;
	border-top-color: transparent;
	border-right-color: transparent;
	border-left-color: transparent;
	border-bottom-color: #e67e22;
	border-bottom-width: 1px;
	border-bottom-radius: 6px;
	border-style: solid;
	color: #FFFFFF;
	padding-bottom: 2px;
	background-color: green;
}</string>
      </property>
      <property name="text">
       <string>Impimer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>30</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="valider_4">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>70</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#valider_4{

background-color: red;
border-radius: 4px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
font-family: century gothic;
font-size: 18px;
color: #a9b7c6;
}
#valider_4:hover{
border-style: solid;
	border-top-color: transparent;
	border-right-color: transparent;
	border-left-color: transparent;
	border-bottom-color: #e67e22;
	border-bottom-width: 1px;
	border-bottom-radius: 6px;
	border-style: solid;
	color: #FFFFFF;
	padding-bottom: 2px;
	background-color: green;
}</string>
      </property>
      <property name="text">
       <string>Trier par prix</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>471</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-family: century gothic;
font-size: 30px;
color: #a9b7c6;</string>
      </property>
      <property name="text">
       <string>Gestion des produits chimiques</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>70</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-width: 1px; border-radius: 4px;
font-size: 18px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;
	color: #a9b7c6;
	background:gray;
	selection-background-color:#007b50;
	selection-color: #FFFFFF;</string>
      </property>
      <property name="placeholderText">
       <string>Donner l'id</string>
      </property>
     </widget>
     <widget class="QPushButton" name="validerC">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>450</y>
        <width>201</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#validerC{

background-color: red;
border-radius: 4px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
font-family: century gothic;
font-size: 18px;
color: #a9b7c6;
}
#validerC:hover{
border-style: solid;
	border-top-color: transparent;
	border-right-color: transparent;
	border-left-color: transparent;
	border-bottom-color: #e67e22;
	border-bottom-width: 1px;
	border-bottom-radius: 6px;
	border-style: solid;
	color: #FFFFFF;
	padding-bottom: 2px;
	background-color: green;
}</string>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="valider_6">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>70</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#valider_6{

background-color: red;
border-radius: 4px;
	border-color: rgb(58, 58, 58);
	border-style: inset;
font-family: century gothic;
font-size: 18px;
color: #a9b7c6;
}
#valider_6:hover{
border-style: solid;
	border-top-color: transparent;
	border-right-color: transparent;
	border-left-color: transparent;
	border-bottom-color: #e67e22;
	border-bottom-width: 1px;
	border-bottom-radius: 6px;
	border-style: solid;
	color: #FFFFFF;
	padding-bottom: 2px;
	background-color: green;
}</string>
      </property>
      <property name="text">
       <string>Chercher</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
